lib/nrf_modem_lib/Kconfig

```diff
@@ -9,6 +9,7 @@ menuconfig NRF_MODEM_LIB
 	imply NET_SOCKETS_OFFLOAD
 	imply NET_SOCKETS_POSIX_NAMES if !POSIX_API
 	select NRF_MODEM
+	default y if BOARD_NRF9160DK_NRF9160_NS
```

Comment:
What about Thingy91? Or any other nRF91-based board, shall we bother? Maybe it'd be better to rely on SoC type instead? Something like:
`default y if SOC_NRF9160_SICA && TRUSTED_EXECUTION_NONSECURE`

Reply:
Yeah, good point.